automated install/basic-install.sh

```diff
@@ -1881,8 +1881,24 @@ FTLdetect() {
 
 }
 
+make_temporary_log() {
+  # Create a random temporary file for the log
+  local TEMPLOG=$(mktemp /tmp/pihole_temp.XXXXXX)
+  # Open handle 3 for templog
+  # https://stackoverflow.com/questions/18460186/writing-outputs-to-log-file-and-console
+  exec 3>"$TEMPLOG"
+  # Delete templog, but allow for addressing via file handle
+  # This lets us write to the log without having a temporary file on the drive, which
+  # is meant to be a security measure so there is not a lingering file on the drive during the debug process
```

Comment:
The comment says `debug`, should be `install` (simple copy-paste error)

Reply:
Thanks, will fix.